app/views/users/shared/_aviso_privacidad.haml

```diff
@@ -0,0 +1,4 @@
+%div
+  %p.text-center
+    No olvides revisar nuestro
+    =link_to "aviso de protección de datos personales.", "http://resguardatos.puebla.gob.mx/images/documentos/AvisoProteccionDatosPers.pdf"
```

Comment:
The = symbol should have one space separating it from code<br>Line is too long. [139/80]<br>Files should end with a trailing newline